package.json

```diff
@@ -18,7 +18,9 @@
     "lint-android": "./android/gradlew -p android spotlessCheck -q",
     "lint": "yarn lint-js && yarn lint-android",
     "release": "yarn prepare && npm login && release-it",
-    "prepare": "bob build && husky install"
+    "prepare": "bob build && husky install",
+    "check-integrity": "node ./scripts/codegen-check-integrity.js",
```

Comment:
"integrity" word does not make much sense in context of what this script is doing. "cohesion" or even better - "consistency" is the word you're looking for.

Reply:
Fixed in https://github.com/software-mansion/react-native-screens/pull/2224/commits/8e4f7a0e7df8b0768a9fc33eecae16c4be3f7a50